server/middleware/workstack.js

```diff
@@ -133,12 +145,23 @@ async function moveTeam(req, res, next) {
 
     if (selected_cases.length > 0 && selected_team) {
         const requests = selected_cases
-            .map(selected => selected.split(':'))
-            .map(([caseId, stageId]) => [`/case/${caseId}/stage/${stageId}/team`, { teamUUID: selected_team }, {
-                headers: User.createHeaders(req.user)
-            }])
-            .map(async options => await sendMoveTeamRequest(req, res, options));
-        await Promise.all(requests);
+            .map(selected => selected.split(':'));
+
+        const updateCaseDataRequests =
+            requests
+                .map(([caseId, stageId]) => [`/case/${caseId}/stage/${stageId}/CaseworkTeamUUID`, { value: selected_team }, {
+                    headers: User.createHeaders(req.user)
+                }])
```

Comment:
From a code-style point of view I'd have either everything on one line or everything on different lines (preferred, as it plays nicer with `diff`) -- but not both:
```suggestion
                .map(([caseId, stageId]) => [
                  `/case/${caseId}/stage/${stageId}/CaseworkTeamUUID`,
                  { value: selected_team },
                  { headers: User.createHeaders(req.user) }
                ])
```

Reply:
Agreed, will change it 👍🏼

Reply:
This has been rectified.